src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -411,6 +414,14 @@ def process_messages(
                 skipped_offsets.add(offset)
                 continue
 
+            if metric_type not in ACCEPTED_METRIC_TYPES:
+                logger.error(
+                    "process_messages.invalid_metric_type",
+                    extra={"org_id": org_id, "metric_type": metric_type, "offset": offset},
+                )
+                skipped_offsets.add(offset)
```

Comment:
in order to identify a message in Kafka you need offset and partition. offset alone is ambiguous as the same offset may and will be reached on all partitions.
This was broken in this consumer but Meredith should have fixed it. 
Now we need to add partitions and offsets in the `skipped_offsets` set.

See: https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/multiprocess.py#L397